### PR TITLE
Mes-1698 Special Needs Fields

### PR DESCRIPTION
**Pull request by Cavalier07**
# Description and relevant Jira numbers

Adds additional fields to the poller so that we can calculate the type of test

## Pull Request checklist

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop

## Sign off process checklist

- [x] Code has been tested manually
- [x] Tested by QA
- [x] PO's approval
- [ ] Reviewers selected in Github
- [x] PR link added to JIRA

### CODE REVIEW

File: schema/exampleSchema.json [hunk 1]
```diff
@@ -50,6 +50,11 @@
           "type": "string",
           "maxLength": 11
         },
+        "vehicleSlotTypeCode": {
+          "description": "A numeric code representing the Vehicle Slot Type",
```
Comment (buzzub) [inline schema/exampleSchema.json]: Is this just a value defined by TARS? Is there a defined list of possible numbers?

Comment (Cavalier07) [reply schema/exampleSchema.json]: There is a page containing all of them - https://wiki.i-env.net/display/MES/Test+Category+Data

Comment (buzzub) [reply schema/exampleSchema.json]: Ah, not a trivial list I see.

Comment (buzzub) [reply schema/exampleSchema.json]: Is the idea that the slot type will be used to help determine the length of the slot then?

Comment (Cavalier07) [reply schema/exampleSchema.json]: We will be using it to show the correct slot length (standard, double, extende) There are a couple of scenarios which only apply to certain vehicle slot types

**Pull request merged**